sycl/include/CL/sycl/detail/pi.h

```diff
@@ -83,41 +81,41 @@ using pi_native_handle = uintptr_t;
 // TODO: populate PI enums.
 //
 typedef enum {
-  PI_SUCCESS = CL_SUCCESS,
-  PI_INVALID_KERNEL_NAME = CL_INVALID_KERNEL_NAME,
-  PI_INVALID_OPERATION = CL_INVALID_OPERATION,
-  PI_INVALID_KERNEL = CL_INVALID_KERNEL,
-  PI_INVALID_QUEUE_PROPERTIES = CL_INVALID_QUEUE_PROPERTIES,
-  PI_INVALID_VALUE = CL_INVALID_VALUE,
-  PI_INVALID_CONTEXT = CL_INVALID_CONTEXT,
-  PI_INVALID_PLATFORM = CL_INVALID_PLATFORM,
-  PI_INVALID_DEVICE = CL_INVALID_DEVICE,
-  PI_INVALID_BINARY = CL_INVALID_BINARY,
-  PI_INVALID_QUEUE = CL_INVALID_COMMAND_QUEUE,
-  PI_OUT_OF_HOST_MEMORY = CL_OUT_OF_HOST_MEMORY,
-  PI_INVALID_PROGRAM = CL_INVALID_PROGRAM,
-  PI_INVALID_PROGRAM_EXECUTABLE = CL_INVALID_PROGRAM_EXECUTABLE,
-  PI_INVALID_SAMPLER = CL_INVALID_SAMPLER,
-  PI_INVALID_BUFFER_SIZE = CL_INVALID_BUFFER_SIZE,
-  PI_INVALID_MEM_OBJECT = CL_INVALID_MEM_OBJECT,
-  PI_OUT_OF_RESOURCES = CL_OUT_OF_RESOURCES,
-  PI_INVALID_EVENT = CL_INVALID_EVENT,
-  PI_INVALID_EVENT_WAIT_LIST = CL_INVALID_EVENT_WAIT_LIST,
-  PI_MISALIGNED_SUB_BUFFER_OFFSET = CL_MISALIGNED_SUB_BUFFER_OFFSET,
-  PI_BUILD_PROGRAM_FAILURE = CL_BUILD_PROGRAM_FAILURE,
-  PI_INVALID_WORK_GROUP_SIZE = CL_INVALID_WORK_GROUP_SIZE,
-  PI_COMPILER_NOT_AVAILABLE = CL_COMPILER_NOT_AVAILABLE,
-  PI_PROFILING_INFO_NOT_AVAILABLE = CL_PROFILING_INFO_NOT_AVAILABLE,
-  PI_DEVICE_NOT_FOUND = CL_DEVICE_NOT_FOUND,
-  PI_INVALID_WORK_ITEM_SIZE = CL_INVALID_WORK_ITEM_SIZE,
-  PI_INVALID_WORK_DIMENSION = CL_INVALID_WORK_DIMENSION,
-  PI_INVALID_KERNEL_ARGS = CL_INVALID_KERNEL_ARGS,
-  PI_INVALID_IMAGE_SIZE = CL_INVALID_IMAGE_SIZE,
-  PI_INVALID_ARG_VALUE = CL_INVALID_ARG_VALUE,
-  PI_INVALID_IMAGE_FORMAT_DESCRIPTOR = CL_INVALID_IMAGE_FORMAT_DESCRIPTOR,
-  PI_IMAGE_FORMAT_NOT_SUPPORTED = CL_IMAGE_FORMAT_NOT_SUPPORTED,
-  PI_MEM_OBJECT_ALLOCATION_FAILURE = CL_MEM_OBJECT_ALLOCATION_FAILURE,
-  PI_LINK_PROGRAM_FAILURE = CL_LINK_PROGRAM_FAILURE,
+  PI_SUCCESS = 0,
+  PI_INVALID_KERNEL_NAME = -46,
```

Comment:
Let's rename all pi_result values different from PI_SUCCESS to start with PI_ERROR:
```suggestion
  PI_ERROR_INVALID_KERNEL_NAME = -46,
```

Reply:
> Let's rename all pi_result values different from PI_SUCCESS to start with PI_ERROR:

OK, will do that.

Reply:
Done in: https://github.com/intel/llvm/pull/6261/commits/9fed447e8d81574fb3355699e0a68e4bb3e6186c